### DISCUSSION TITLE
How do I change the keyboard shortcut to open the assistant?

### DISCUSSION THREAD

**Discussion by twelvemp** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#368] [Q&A] [answered]
How do I change the keyboard shortcut to open the assistant?
I don't see it anywhere in the settings and I'm not smart enough to change it on my own without knowing how. Thanks!!

**Comment by sumeshir26** [comment, marked as answer]
Hi @twelvemp,
This feature has not been released yet in the release version and can be used by [building the project yourselves](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#how-to-build).
